applications/tari_base_node/src/parser.rs

```diff
@@ -179,6 +182,9 @@ impl Parser {
             ListHeaders => {
                 println!("List the last headers up to a maximum of 10 of the current chain");
             },
+            EnableMining => {
```

Comment:
How is this compiling?

Reply:
Intersting.
```
 EnableMining  => {
    |------------ matches any value
```
which is very misleading, but basically, this match statement is very broken.

Reply:
mmm interesting, I specifically tried to run a full tests at this to capture stuff like this...

Reply:
updated